container/docker/factory.go

```diff
@@ -197,24 +197,38 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo, ignoreMetrics c
 	)
 
 	if dockerStorageDriver == devicemapperStorageDriver {
-		// If the storage drive is devicemapper, create and start a
-		// ThinPoolWatcher to monitor the size of container CoW layers with
-		// thin_ls.
-		dockerThinPoolName, err := dockerutil.DockerThinPoolName(*dockerInfo)
-		if err != nil {
-			return fmt.Errorf("couldn't find device mapper thin pool name: %v", err)
-		}
-
-		dockerMetadataDevice, err := dockerutil.DockerMetadataDevice(*dockerInfo)
-		if err != nil {
-			return fmt.Errorf("couldn't determine devicemapper metadata device")
+		_, err := devicemapper.ThinLsBinaryPresent()
```

Comment:
There's an issue to track dependencies of kubelet, and I've called it out there.  we neeed to figure out the scheme in general for tracking and verifying stuff like that.